sections/appendix/multi_process_gibbs.tex

```diff
@@ -0,0 +1,7 @@
+\subsection{Parallel Gibbs Sampling}\label{sec:appendix_para_gibbs}
+We have also implemented a Gibbs sampler, which works in parallel by splitting up the dataset into $p$ parts, where $p$ is the number of processes.
+This is to create $p$ amount of progress for each process and then combine them.
+Each process gets a specific split of the dataset and the available words. 
+This is done to avoid race conditions on increasing and decreasing counts in the Gibbs sampler, which is explained \autoref{sec:appendix_gibbs}.

```

Comment:
```suggestion
This is done to avoid race conditions on increasing and decreasing counts in the Gibbs sampler, which is explained in \autoref{sec:appendix_gibbs}.
```